### PR TITLE
Add `no-meta-redirect-with-time-limit` rule

### PR DESCRIPTION
**Pull request by MelSumner**
When merged, this PR will provide a check to prevent the use of a timed delay on `<meta http-equiv="refresh"/>' where the `content` delay value is greater than 0.

### CODE REVIEW

File: lib/rules/lint-no-meta-redirect-with-time-limit.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+      ElementNode(node) {
+        if (AstNodeInfo.hasAttribute(node, 'hidden')) {
+          return;
+        }
+
+        if (AstNodeInfo.hasAttributeValue(node, 'aria-hidden', 'true')) {
+          return;
+        }
+
+        const isMeta = AstNodeInfo.isElementNode(node, 'meta');
```
Comment (lifeart) [inline lib/rules/lint-no-meta-redirect-with-time-limit.js]: `isElementNode` accept only one argument

```suggestion
        const isMeta = node.tag === 'meta';
```

Is it possible to move this assert on the top because of perf reasons? (we don't need to check element visibility if this is not `meta` node.

File: lib/rules/lint-no-meta-redirect-with-time-limit.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        const hasMetaRedirect = AstNodeInfo.hasAttribute(node, 'http-equiv');
+
+        if (isMeta && hasMetaRedirect) {
+          const contentAttr = AstNodeInfo.hasAttribute(node, 'content');
+          const contentAttrValue = AstNodeInfo.elementAttributeValue(node, 'content');
+          if (contentAttr) {
+            if (contentAttrValue.charAt(0) > 0) {
```
Comment (lifeart) [inline lib/rules/lint-no-meta-redirect-with-time-limit.js]: I think it could be simplyfied to 
```suggestion
            if (!AstNodeInfo.hasAttributeValue(node, 'content', '0')) {
```

File: lib/rules/lint-no-meta-redirect-with-time-limit.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+      message,
+      line: node.loc && node.loc.start.line,
+      column: node.loc && node.loc.start.column,
+      source: this.sourceForNode(node),
+    });
+  }
+  visitor() {
+    return {
+      ElementNode(node) {
+        if (AstNodeInfo.hasAttribute(node, 'hidden')) {
```
Comment (rwjblue) [inline lib/rules/lint-no-meta-redirect-with-time-limit.js]: Does this attribute actually have effect on `meta`? Aren't they always hidden (AFAICT they have no display)?

File: lib/rules/lint-no-meta-redirect-with-time-limit.js [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+          return;
+        }
+
+        if (AstNodeInfo.hasAttributeValue(node, 'aria-hidden', 'true')) {
```
Comment (rwjblue) [inline lib/rules/lint-no-meta-redirect-with-time-limit.js]: Same here, what does this have to do with `meta`? I don't see `hidden` or `aria-hidden` mentioned in the references...

File: lib/rules/lint-no-meta-redirect-with-time-limit.js [hunk 5]
```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+const AstNodeInfo = require('../helpers/ast-node-info');
+const Rule = require('./base');
+
+module.exports = class noMetaRedirectWithTimeLimit extends Rule {
```
Comment (rwjblue) [inline lib/rules/lint-no-meta-redirect-with-time-limit.js]:
```suggestion
module.exports = class NoMetaRedirectWithTimeLimit extends Rule {
```
